content/04.results.md

```diff
@@ -13,8 +13,11 @@ A) The distribution of unique participant samples across short histologies and i
 ### Landscape of Genomic Alterations
 
 The oncoprint will provide a visualization of the genomic alterations found in the analyses implemented throughout the OpenPBTA project.
+The genes displayed on the oncoprint have been filtered to include the top mutated genes and to the top genes with recurrent CNVs.
+The samples displayed have been filtered to include independent samples of primary and secondary specimens with whole genome sequencing (WGS) data.
```

Comment:
I would move this detail into the figure legend, instead. Some comments on the wording in the suggestion below.